### PR TITLE
run_tasks: improve sentry reporting

### PR DESCRIPTION
**Pull request by jdurgin**
Correct the url generated, and include additional tags for filtering by suite/machine_type/os.

### CODE REVIEW

File: teuthology/run_tasks.py [hunk 1]
```diff
@@ -111,10 +111,11 @@ def run_tasks(tasks, ctx):
...
             }
-            if 'teuthology_branch' in config:
-                tags['teuthology_branch'] = config['teuthology_branch']
-            if 'branch' in config:
-                tags['branch'] = config['branch']
+            optional_tags = ('teuthology_branch', 'branch', 'suite',
+                             'machine_type', 'os_type', 'os_version')
+            for tag in optional_tags:
+                if tag in config:
+                    tags[tag] = config[tag]
```
Comment (kshtsk) [inline teuthology/run_tasks.py:118]: Why do not use standard teuthology deep_merge and invent something new?

File: teuthology/run_tasks.py [hunk 2]
```diff
@@ -111,10 +111,11 @@ def run_tasks(tasks, ctx):
                 'task': taskname,
                 'owner': ctx.owner,
             }
-            if 'teuthology_branch' in config:
-                tags['teuthology_branch'] = config['teuthology_branch']
-            if 'branch' in config:
-                tags['branch'] = config['branch']
+            optional_tags = ('teuthology_branch', 'branch', 'suite',
+                             'machine_type', 'os_type', 'os_version')
+            for tag in optional_tags:
```
Comment (kshtsk) [inline teuthology/run_tasks.py:116]: This change looks unrelated to the PR name.

File: teuthology/run_tasks.py (hunk 1 above)
Comment (jdurgin) [reply teuthology/run_tasks.py:118]: it's doing something different - we're only adding a subset of values from the config. deep_merge would include all of them.

File: teuthology/run_tasks.py (hunk 2 above)
Comment (jdurgin) [reply teuthology/run_tasks.py:116]: changed the title after I added a 2nd commit

File: teuthology/run_tasks.py (hunk 1 above)
Comment (kshtsk) [reply teuthology/run_tasks.py:118]: yeah... I see it now

**Review by badone — APPROVED** [review]
